server/instrumentation-backend/src/main/java/sh/calaba/instrumentationbackend/query/ast/UIQueryUtils.java

```diff
@@ -310,9 +310,16 @@ public static String getTag(View view) {
 	}
 
 	public static int[] getViewLocationOnScreen(View view) {
-		ViewWrapper viewWrapper = new ViewWrapper(view);
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.ICE_CREAM_SANDWICH_MR1 && Build.VERSION.SDK_INT < 28) {
```

Comment:
Do we know what version the bug was fixed? in that case I would prefer to use that instead of 28